server/src/main/java/org/elasticsearch/index/mapper/AbstractScriptFieldType.java

```diff
@@ -296,5 +299,9 @@ protected final Script getScript() {
             }
             return script.get();
         }
+
+        protected final boolean onErrorContinue() {
```

Comment:
I have been wondering if we should carry around a boolean, or rather an enum. I see we use a string for index time scripted fields. Hopefully adopting an enum clarify things without adding much complexity?

Reply:
This should be an enum now.